skbio/metadata/_interval.py

```diff
@@ -726,6 +765,50 @@ def __repr__(self):
 
         return '\n'.join([l1, l2] + items)
 
+    @experimental(as_of='0.5.0-dev')
+    def __copy__(self):
+        '''Return a shallow copy.
+
+        Notes
+        -----
+        The references to original ``Interval`` objects are created
```

Comment:
I don't think this is true anymore. Now deep and shallow copy will always create a new `IntervalMetadata` object with new `Interval` objects. The metadata dict is the only thing that is shallow vs. deep copied.

Can you update this docstring, as well as the deepcopy docstring, to describe this behavior? Something along the lines of:

(For shallow copy)

"The `IntervalMetadata` copy will have copies of the `Interval` objects present in this `IntervalMetadata` object. Each `Interval` object's `.metadata` dictionary will be a shallow copy."

(For deep copy)

"The `IntervalMetadata` copy will have copies of the `Interval` objects present in this `IntervalMetadata` object. Each `Interval` object's `.metadata` dictionary will be a deep copy."